docs/src/_includes/layouts/doc.html

```diff
@@ -48,20 +47,24 @@ <h2 id="resources">Resources</h2>
 
     <div class="docs-content">
         <main id="main" tabindex="-1" class="docs-main">
-            <h1>{{ title }}</h1>
-            {% include 'components/docs-toc.html' %}
-
-            {{ all_content | safe }}
+            <div class="docs-main__content">
+                <h1>{{ title }}</h1>
+                {% include 'components/docs-toc.html' %}
+                {{ all_content | safe }}
+            </div>
 
             <div class="docs-edit-link">
                 <a href="{{ edit_link }}" class="c-btn c-btn--secondary">Edit this page</a>
             </div>
         </main>
 
-        <div class="docs-right-sidebar">
-            <aside class="docs-ad">
-                {%- include 'partials/carbon-ad.html' -%}
-            </aside>
+        <div class="docs-aside">
+            <div class="docs-aside__content">
+                <aside class="docs-ad">
+                    {%- include 'partials/carbon-ad.html' -%}
+                </aside>
+                {% include 'components/docs-toc.html' %}
```

Comment:
This will add duplicate ids `js-toc-label` and `js-toc-panel`, and we have a script that targets these ids, so it might be better to fix this to avoid unexpected behavior in different browsers.

Reply:
I think we should hold off on dealing with this now. Let's just get the current prototype version deployed and then we can see if this is actually causing any problems and narrowly focus on solving it.

Reply:
Agreed. Everything else LGTM, so I'll merge this now to be deployed.